app/Repositories/Summit/DoctrineSummitSponsorshipAddOnRepository.php

```
@@ -83,6 +84,18 @@ protected function getBaseEntity(): string
      */
     public function getMetadata(Summit $summit): array
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$summit'. (undefined)

(UnusedFormalParameter)